projects/canopy/src/styles/mixins.scss

```diff
@@ -19,7 +19,8 @@
   $inner-bg-color: var(--default-inner-focus-color)
 ) {
   outline: 0;
-  box-shadow: 0 0 0 0.125rem $inner-bg-color, 0 0 0 0.375rem $bg-color;
+  box-shadow: 0 0 0 var(--innner-focus-width) $inner-bg-color,
```

Comment:
Typo: `--innner-focus-width`

Reply:
Webstorm doesn't seem to spot typos in css variables like it does in other scenarios :-p